t/t5318-commit-graph.sh

```diff
@@ -415,9 +409,10 @@ corrupt_graph_and_verify() {
 }
 
 test_expect_success POSIXPERM,SANITY 'detect permission problem' '
+	test_when_finished chmod 666 $objdir/info/commit-graph &&
```

Comment:
This seems necessary on the OSX platform due to the `chmod 000` below. This test caused failures down the road.